Source/WebKit/NetworkProcess/soup/NetworkProcessMainSoup.cpp

```diff
@@ -49,9 +49,12 @@ class NetworkProcessMainSoup final: public AuxiliaryProcessMainBaseNoSingleton<N
 
     void platformFinalize() override
     {
-        // FIXME: Is this still needed? We should probably destroy all existing sessions at this point instead.
-        // Needed to destroy the SoupSession and SoupCookieJar, e.g. to avoid
-        // leaking SQLite temporary journaling files.
+        Vector<PAL::SessionID> sessionIDs;
+        process().forEachNetworkSession([&sessionIDs](auto& session) {
+            sessionIDs.append(session.sessionID());
+        });
+        for (auto& sessionID : sessionIDs)
+            process().destroySession(sessionID);
         process().destroySession(PAL::SessionID::defaultSessionID());
```

Comment:
I guess the default session has already been destroyed at this point.

Reply:
It should be, you are right